person.rb

```diff
@@ -7,9 +7,10 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] It required you to create a method for adding a rental with the person sent as a parameter to create a new rental achieving a has-many association 😄

Reply:
Added. Thanks for the feedback